src/proguard-android/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'com.android.application' version '8.7.0'
+}
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+android {
+    namespace 'com.microsoft.proguard.android'
+    // Setting the minimum we support at the moment, might not matter
+    compileSdk 21
```

Comment:
aren't we on 23 now?

Reply:
The runtime is still 21 right now, but I don't think their gradle task even looks at this number:

* https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:build-system/gradle-core/src/main/java/com/android/build/gradle/ProguardFiles.java;l=116

I get a build error if you don't list some number, though:

```
A problem occurred configuring root project 'proguard-android'.
> compileSdkVersion is not specified. Please add it to build.gradle
```